src/utils/format-numbers.js

```diff
@@ -64,6 +64,18 @@ export function formatBytes(bytes: number): string {
   }
 }
 
+export function formatSI(num: number): string {
+  if (num < 10000) {
+    return formatNumber(num);
+  } else if (num < 10000000) {
+    return formatNumber(num / 1000) + 'K';
+  } else if (num < 10000000000) {
+    return formatNumber(num / 1000000) + 'M';
+  } else {
```

Comment:
I don't understand the logic completely here: why don't you start the `K` at 1000 or 10000, etc ?

Reply:
Please add some comment explaining the logic, at least :)